src/app/hm-admin/services/models/PagedCollection.ts

```diff
@@ -0,0 +1,22 @@
+export class PagedCollection {
+
+  public context;
+  public id;
+  public type;
+  public firstPage;
+  public itemsPerPage;
+  public lastPage;
+  public members;
+  public totalItems;
+
+  constructor(properties: any = {}, definition: any = {}) {
```

Comment:
Since we are assuming that properties has some well known properties of its own ('@context', '@id', etc.). We should have a typescript interface for it.